### PR TITLE
Fix folder name 'avatarify' to 'avatarify-python'

### PR DESCRIPTION
**Pull request by christianll9**

### CODE REVIEW

File: docs/README.md [hunk 1]
```diff
@@ -132,7 +132,7 @@ Docker images are only availabe on Linux.
 3. Clone `avatarify` and install its dependencies (v4l2loopback kernel module):
 ```bash
 git clone https://github.com/alievk/avatarify-python.git
-cd avatarify
+cd avatarify-python
 bash scripts/install_docker.sh
 ```
 4. Build the Dockerfile:
```
Comment (JohanAR) [inline docs/README.md:138]: There's a "cd avatarify" on line 140 in docker instructions, but it seems like I can't comment lines that are too far from your changes

**Review by JohanAR — CHANGES_REQUESTED** [review]
Good changes! Please change line 140 too

There are also a few instances of "replace this with your actual install path" on line 173 and line 195 that I think would be more clear if they were changed avatarify-python

File: docs/README.md (hunk 1 above)
Comment (christianll9) [reply docs/README.md:138]: Okay, I did this now

**Review by JohanAR — APPROVED** [review]
Thanks!

**Pull request merged**